### PR TITLE
Hotfix/#8 no reload

### LINKED ISSUE #8: Changing URL is only reflected after restart
As an admin, I want to be able to modify the target URL for the search at runtime

## Expected behavior
The search requests are being routed to the new URL immediately

## Actual behavior
Request is routed to the old URL until the system is restartet

### PR DESCRIPTION
**Pull request by mrsimpson**
fixes #8

### CODE REVIEW

File: client/template/search.js [hunk 1]
```diff
@@ -39,6 +39,7 @@ Template.ChatpalSearch.onRendered(function() {
 				console.log('ping failed');
 				this.enabled.set(false);
 			} else {
+				this.enabled.set(true);
```
Comment (mrsimpson) [inline client/template/search.js:42]: Once a result could be retrieved, this reactive var needs to be reset in order to make the results being displayed

File: server/service/bot_service.js [hunk 2]
```diff
@@ -109,21 +109,8 @@ class ChatpalBotService {
...
-		console.log(`Initialize Chatpal Bot Service with url ${ v.value }`);
-		Chatpal.service.BotService .setBaseUrl(v.value);
-	},
-	changed(n, v) {
-		//TODO is this a bug
-		if (!v.value) { return; }
-		console.log(`Re-Initialize Chatpal Bot Service with url ${ v.value }`);
-		Chatpal.service.BotService .setBaseUrl(v.value);
-	},
-	removed() {
```
Comment (mrsimpson) [inline server/service/bot_service.js:123]: A setting should never be removed at runtime, only in a migration

File: server/service/bot_service.js [hunk 3]
```diff
@@ -109,21 +109,8 @@ class ChatpalBotService {
...
-		console.log('Stop Chatpal Bot Service');
-		Chatpal.service.BotService .setBaseUrl(undefined);
-	}
+RocketChat.settings.get('CHATPAL_BOT_BASEURL', (id, value)=>{
```
Comment (mrsimpson) [inline server/service/bot_service.js:112]: `get(callback)` is the best trigger to use for invalidation according to rodrigok from RocketChat

File: server/service/bot_service.js [hunk 4]
```diff
@@ -109,21 +109,8 @@ class ChatpalBotService {
...
+	Chatpal.service.BotService.setBaseUrl(value);
```
Comment (mrsimpson) [inline server/service/bot_service.js:113]: Was there any purpose to have the "blank" before the method's name?

**Review by mrsimpson — COMMENTED** [review]
Some minor changes with major effects ;)

**Review by ja-fra — APPROVED** [review]

**Pull request merged**